docs/_includes/common/lightbox-dialog.html

```diff
@@ -305,22 +317,32 @@ <h3 id="lightbox-alert">Alert Lightbox</h3>
         <div class="demo__inner">
             <button class="btn btn--primary dialog-lightbox-button" data-makeup-dialog-button-for="lightbox-dialog-alert-0" type="button">Open Alert Dialog</button>
             <div aria-labelledby="lightbox-dialog-alert-title" aria-modal="true" class="lightbox-dialog lightbox-dialog--mask-fade" data-makeup-dialog-has-transitions="true" hidden id="lightbox-dialog-alert-0" role="alertdialog">
```

Comment:
Nitpick, but can you remove the `data-makeup-dialog-has-transitions="true"` from the visible example code, and any other occurrences you might see.

Reply:
Done